docs/product-user-guide/src/main/asciidoc/service-start-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='service-start-proc_{context}']
+
+= Starting a service in {KIE_SERVER}
```

Comment:
For the scope of this adoc file, my jargon here was "Kie Container". Or could do also "decision service".